scripts/bootstrap.sh.in

```diff
@@ -78,37 +130,53 @@ run_bootstrap_prechecks() {
 }
 
 install_python_kubernetes_client() {
-    ${SALT_CALL} --local --retcode-passthrough state.apply metalk8s.python-kubernetes saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
+    "$SALT_CALL" --local --retcode-passthrough state.apply \
```

Comment:
Why quote it ? if we want to be able to add option to the variable we should not quote it

Reply:
If we want to add options, we should rather use an array.
We should not rely on the expansion of a string to do that, it's a horrible thing.